package.json

```diff
@@ -12,7 +12,7 @@
   "peerDependencies": {
     "prop-types": "^15.5.10",
     "react": ">=16.3.0",
-    "react-native": ">=0.54.0"
+    "react-native": "0.69.4"
```

Comment:
Can you use a looser definition here e.g. a compatible version range or a `>=`?

Reply:
done 👍